test/conformance/tests/cilium-connectivity-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+ns=cilium-test
+
+# Grab the path to the kubeconfig file.
+export KUBECONFIG=$(terraform output --raw path_to_kubeconfig_file)
+
+# Run the connectivity tests.
+kubectl -n kube-system port-forward svc/hubble-relay 4245:80 &
+# NS precreation is required because of https://www.talos.dev/v1.5/kubernetes-guides/configuration/pod-security/
+kubectl create ns ${ns}
+kubectl label ns ${ns} pod-security.kubernetes.io/enforce=privileged
+kubectl label ns ${ns} pod-security.kubernetes.io/warn=privileged
+PID=$!
+set +e
```

Comment:
```suggestion
```

We need to remove this; otherwise, a `cilium connectivity test` non-zero exit code won't be caught.